docs/avro.rst

```diff
@@ -127,13 +76,17 @@ field, which is missing in the old data.
     in the Apache Avro project to understand how, for data that was encoded with an older schema, Avro decodes that
     data with a newer, backward-compatible schema.
 
+If the compatibility is set to ``BACKWARD_TRANSITIVE`` (not just ``BACKWARD``), then it means that consumers using the new schema can read data written by producers using all previously registered schemas (not just the latest schema).
+
 
 .. _avro-forward_compatibility:
 
 Forward Compatibility
 ^^^^^^^^^^^^^^^^^^^^^
 
-Forward compatibility means that data encoded with a newer schema can be read with an older schema.
+``FORWARD`` compatibility means that data encoded with a newer schema can be read with an older schema.
```

Comment:
How about this: FORWARD compatible schema means that version X-1 of the schema can be used to decode data encoded with version X of the schema, but not necessarily version X-2 and before.

Reply:
Good idea. I've adapted this suggestion

Reply:
Good idea.  I've adapted this suggestion.